src/com/owncloud/android/utils/FileStorageUtils.java

```diff
@@ -126,6 +127,30 @@ public static String getInstantUploadFilePath(Context context, String fileName)
     }
 
     /**
+     * Returns account for instant upload or null, if not defined.
+     * @return instant upload account or null
+     */
+    public static Account getInstantUploadAccount(Context context) {
+        return getAccount(context, "instant_upload_path_account");
+    }
+
+    /**
+     * Returns account for instant video upload or null, if not defined.
+     * @return instant video upload account or null
+     */
+    public static Account getInstantVideoUploadAccount(Context context) {
+        return getAccount(context, "instant_video_upload_path_account");
+    }
+
+    private static Account getAccount(Context context, String prefName) {
+        SharedPreferences pref = PreferenceManager.getDefaultSharedPreferences(context);
+        String accountName = pref.getString(prefName, null);
+        Account account = AccountUtils.getOwnCloudAccountByName(MainApp.getAppContext(),
+                accountName);
+        return account;
+    }
+    
```

Comment:
I think it makes more sense to add these new methods to `AccountUtils` class, dont you think?